src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheckTest.java

```diff
@@ -738,8 +739,8 @@ public void testNoPackage2() throws Exception {
         final String[] expected = {
             "7: " + getCheckMessage(MSG_LINE_SEPARATOR,
                 "com.sun.accessibility.internal.resources.*"),
-            "11: " + getCheckMessage(MSG_LINE_SEPARATOR, "java.util.Arrays"),
-            "19: " + getCheckMessage(MSG_LINE_SEPARATOR,
+            "11: " + getCheckMessage(MSG_SEPARATED_IN_GROUP, "java.util.Arrays"),
+            "19: " + getCheckMessage(MSG_SEPARATED_IN_GROUP,
```

Comment:
1) please add test case from https://github.com/checkstyle/checkstyle/pull/7364#issuecomment-564741290 to our Input code base.

2) please add more test cases for different "customImportOrderRules", as right now we test it only on "STATIC###THIRD_PARTY_PACKAGE", please add few tests with different values for this property to make sure logic works fine for them too.

3) please add test input to `it` test part for google style coverage, as change affects behavior of of google style coverage. Inputs can be copied from `ut`.

Reply:
Tests added.